src/logger/Log.ts

```diff
@@ -1,13 +1,11 @@
 // Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-import LogLevel from './LogLevel';
-
 export default class Log {
   constructor(
     public sequenceNumber: number,
     public message: string,
     public timestampMs: number,
-    public logLevel: LogLevel
+    public logLevel: string
```

Comment:
You can keep it typed all the way through. When you go to compose the log string, convert the `logLevel: LogLevel` with `LogLevel[logLevel]` to get the string.